bindings/python/test/test_numpy.py

```diff
@@ -45,13 +48,14 @@ def setUp(self):
         self.cmd_listener.reset()
         self.getmore_listener.reset()
 
-    def assert_numpy_equal(self, actual, expected):
+    def assert_numpy_equal(self, actual, expected, schema=None):
         self.assertIsInstance(actual, dict)
-        for field in self.schema:
+        for field in schema or self.schema:
```

Comment:
Do we need to pass the schema here at all? What about just `for field in expected:`?